src/SixLabors.Fonts/Font.cs

```diff
@@ -167,6 +167,13 @@ public IEnumerable<Glyph> GetGlyphs(CodePoint codePoint, TextAttributes textAttr
             }
         }
 
+        /// <summary>
+        /// Gets the unicode codepoints for which a glyph exists in the font.
+        /// </summary>
+        /// <returns>An enumerable containing all available codepoints.</returns>
+        public IEnumerable<CodePoint> GetAvailableCodePoints()
```

Comment:
This should probably be IReadOnlyCollection<CodePoint> since we want it to be a one-time enumeration.